ironfish/src/network/peerNetwork.ts

```diff
@@ -33,7 +33,6 @@ import {
   GetBlockTransactionsResponse,
 } from './messages/getBlockTransactions'
 import { GetCompactBlockRequest, GetCompactBlockResponse } from './messages/getCompactBlock'
-import { GossipNetworkMessage } from './messages/gossipNetworkMessage'
```

Comment:
Not related to this code change, but we should bump the min protocol version to 18 to indicate we're not supporting the older messages any longer (even though there aren't many people left on versions earlier than v0.1.45)

Reply:
Done 👍🏻